packages/itwinui-css/src/bridge.scss

```diff
@@ -291,16 +327,28 @@
   /* ---------------------------------------------------------------------------- */
   /* Input */
 
-  .iui-input:where(:not([role='combobox'])) {
+  .iui-input {
     // StrataKit input background colors are different from global values
     --_iui-field-color-background: var(--stratakit-color-bg-control-textbox);
-    --_iui-field-color-background-hover: var(--_iui-field-color-background);
-    --_iui-field-color-background-disabled: var(--_iui-field-color-background);
+    --_iui-field-color-background-hover: color-mix(
+      in oklch,
+      var(--stratakit-color-bg-control-textbox) 100%,
+      var(--stratakit-color-glow-hue) var(--stratakit-color-bg-glow-base-hover-\%)
+    );
+    --_iui-field-color-background-disabled: var(--stratakit-color-bg-control-textbox);
   }
 
   .iui-input-flex-container {
     // StrataKit input background colors are different from global values
-    background-color: var(--stratakit-color-bg-control-textbox);
+    --_iui-input-background-color: var(--stratakit-color-bg-control-textbox);
+
+    &:where(:hover) {
+      --_iui-input-background-color: color-mix(
+        in oklch,
+        var(--stratakit-color-bg-control-textbox) 100%,
+        var(--stratakit-color-glow-hue) var(--stratakit-color-bg-glow-base-hover-\%)
+      );
+    }
   }
 
   .iui-input-flex-container:where([data-iui-disabled='true']) {
```

Comment:
Can this be removed now that `--_iui-input-background-color` is being set above?

Reply:
When removing `.iui-input-flex-container:where([data-iui-disabled='true'])` it does inherit the hover state background-color change.

Reply:
Got it https://github.com/iTwin/iTwinUI/pull/2598/commits/113f17cc2a797145622cbde0db78f6a2e91572cf